### PR TITLE
Fix dead link in quickstart.md

### PR DESCRIPTION
**Pull request by vdaubry**
#### :rocket: Why this change?

Dead link in documentation
#### :memo: Related issues and Pull Requests

Fix Ruby on Rails example application link

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ Dredd's [hooks](hooks.md) enable you to write some glue code in your favorite la
 For more complex example applications, please refer to:
 
 - [Express.js example application](http://github.com/apiaryio/dredd-example)
-- [Ruby on Rails example application](https://github.com/theodorton/dredd-test-rails)
+- [Ruby on Rails example application](https://github.com/apiaryio/dredd-test-rails)
```
Comment (honzajavorek) [inline docs/quickstart.md]: @theodorton moved the repo to https://gitlab.com/theodorton/dredd-test-rails. @netmilk do you know which one of the repos (https://gitlab.com/theodorton/dredd-test-rails vs https://github.com/apiaryio/dredd-test-rails) is the one we should recommend in docs?

**Comment by honzajavorek** [comment]
Thanks!

File: docs/quickstart.md (hunk 1 above)
Comment (theodorton) [inline docs/quickstart.md]: Hey :) Sorry for breaking this. I've put the repo back up.

Comment (honzajavorek) [inline docs/quickstart.md]: Ha, no problem! I assumed it's easier to fix the link than to maintain repo mirror :) Just wanted to know if the apiary repo is some sort of backup fork of the original repo or what. Not sure if it contains any differences from your repo, will have to check.

**Comment by honzajavorek** [comment]
Since https://github.com/theodorton/dredd-test-rails now exists again on the original link, I think we can close this now. @vdaubry thanks for bringing this up! @theodorton thanks for reinstantiating the repo :)

**Pull request closed without merging**